test/stdlib/UnsafeRawBufferPointer.swift

```diff
@@ -75,6 +75,59 @@ UnsafeRawBufferPointerTestSuite.test("initFromArray") {
   expectEqual(array2, array1)
 }
 
+UnsafeRawBufferPointerTestSuite.test("initializeMemory(as:from:).underflow") {
```

Comment:
I'm confused by the underflow terminology here. This sequence still overflows the buffer right? Isn't it an overflow with underestimated count?

I don't see a test case for actual underflow!